client/webserver/site/src/html/markets.tmpl

```diff
@@ -527,32 +527,31 @@
                   <span data-tmpl="tooltip" class="ico-info me-2" data-tooltip="Prevent temporary overlocking of funds"></span>
                   <img data-tmpl="chainIcon" class="micro-icon me-1">
                 </div>
+              </div>
+            </div>
 
-                <div id="booleanOptTmpl" class="d-flex justify-content-between align-items-center fs14 ps-2">
```

Comment:
I just moved the `rangeOptTmpl` and the `boolOptTmpl` to be outside of the `orderOptTmpl` here, there are no other changes.